basic.py

```diff
@@ -103,11 +104,11 @@ def sub_event_catcher(event):
         last_events[event.meta['host']] = event
 
 
-@match(regex=r'^s([^ a-zA-Z0-9])(.*?)\1(.*?)(?:\1(.+?))?$')
+@match(regex=SED_REGEX)
 def sub_match(event, sep, pattern, replacement, flags, *args, **kwargs):
     """Matches any substitute string (as sed would), applies this substitution to
-    the user's last message and sents it back to processing.
-    """
+    the user's last message and sends it back to processing."""
```

Comment:
@mrshu actually, as per [PEP257](https://www.python.org/dev/peps/pep-0257) (Docstring conventions):
"Unless the entire docstring fits on a line, place the closing quotes on a line by themselves."